src/services/users/signin-user-by-email.test.js

```diff
@@ -0,0 +1,51 @@
+const { expect } = require('chai')
+const db = require('../../db')
+const {fixtures} = require('../../../test/utils')
+const signInUserByEmail = require('./signin-user-by-email')
+
+describe('User signin', () => {
+	let user
+	let validPassword
+	beforeEach(async () => {
+		user = await fixtures.insertUser({
+			password : 'validPaswword123$'
+		})
+		validPassword = 'validPaswword123$'
+	})
+
+	it('should throw an error if user does not exist', async () => {
+		const { id,email } = user
+		await db.query(
+			`DELETE FROM users
+             WHERE id = $1`, [id])
+
+		return expect(signInUserByEmail(email , validPassword))
+			.to.be.rejectedWith(
+				'Invalid email or password.')
+	})
+
+	it('should return the right response', async () => {
+		const { id, email} = user
+		await signInUserByEmail(
+			email, validPassword)
+
+		const { rows } = await db.query(
+			`SELECT * FROM users
+             WHERE id = $1`, [id])
+
+		const  loggedInUser = rows[0]
+
+		return expect(loggedInUser).to.include({
```

Comment:
the expectation  is still going to always pass, 

we inserted a user, called the signin function, and got the same user from the database, so nothing changes

if you're testing if it returns the right response, you should compare the response of the function to your user if you want